### PR TITLE
chore: migrate to new prow reviewing

### PR DESCRIPTION
**Pull request by wuhuizuo**
- Backgroup: https://internals.tidb.io/t/topic/854
- Ref: https://github.com/ti-community-infra/configs/issues/824

Signed-off-by: wuhuizuo <wuhuizuo@126.com>

### What problem does this PR solve?

Issue Number: ref https://github.com/ti-community-infra/configs/issues/824

Problem Summary:

### Check List

Tests 

- [x] No code

Side effects

- None

Documentation

- None

### Release note

```release-note
None
```

### CODE REVIEW

File: OWNERS
Comment (zanmato1984) [inline OWNERS:1]: This list is out-of-date with the latest version: https://github.com/pingcap/community/blob/master/teams/tiflash/membership.json

File: OWNERS [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  - solotzg
+  - windtalker
+  - wshwsh12
+  - XuHuaiyu
+  - yibin87
+  - ywqzzy
+reviewers:
+  # - tiflash-reviewers
+  - LiSong0214
+  - ywqzzy
```
Comment (wuhuizuo) [inline OWNERS:31]:
```suggestion
  - xzhangxian1008
  - ywqzzy
```

File: OWNERS
Comment (wuhuizuo) [reply OWNERS:1]: updated.

**Review by zanmato1984 — APPROVED** [review]
LGTM

**Review by SeaRise — APPROVED** [review]

**Pull request merged**